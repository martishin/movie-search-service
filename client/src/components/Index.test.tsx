import { cleanup, render } from "@testing-library/react";
import Index from "./Index";
import { expect } from "@jest/globals";

afterEach(() => {
  cleanup();
});
describe("Index", () => {
  test("should display Hello world!", () => {
    const { container, getByText } = render(<Index />);
    expect(container).toMatchSnapshot();

    expect(getByText("Hello world!")).toBeTruthy();
  });
});
