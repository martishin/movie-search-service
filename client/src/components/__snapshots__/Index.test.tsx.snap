// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should display Hello world! 1`] = `
<div>
  <h1
    class="text-3xl font-bold underline"
  >
    Hello world!
  </h1>
</div>
`;
